src/monorepo-shipit/src/RepoGit.js

```diff
@@ -246,7 +258,9 @@ export default class RepoGit implements AnyRepo, SourceRepo, DestinationRepo {
     } else {
       const diff = this.renderPatch(changeset);
       try {
-        this._gitCommand('am', '--keep-non-patch', '--keep-cr').setStdin(diff).runSynchronously();
+        this._gitCommand('am', '--keep-non-patch', '--keep-cr', '--committer-date-is-author-date')
```

Comment:
`--committer-date-is-author-date` ensures that the commit date is the same as the author date (instead of the timestamp of the commit). This results in a nicer commit history when syncing a large repo for the first time, instead of everything being the same commit timestamp they're the appropriate:

<img width="467" alt="image" src="https://github.com/adeira/universe/assets/6801309/a1bca467-5c3a-4781-b535-439ad3a4a632">

Reply:
And here is a subsequent run, works great: 

Source:
<img width="580" alt="image" src="https://github.com/adeira/universe/assets/6801309/25d31f49-09c3-423b-a41f-1034c8bcf6c3">

Destination:
<img width="580" alt="image" src="https://github.com/adeira/universe/assets/6801309/858f4955-4df5-4426-b0ab-e9f3c78ba200">

Reply:
This seems to be a great addition; thanks! 👍